portfolio/src/main/webapp/script.js

```diff
@@ -87,16 +87,39 @@ function changePattern(pattern) {
   }
 }
 
-async function retrieveComments() {
```

Comment:
We don't normally encourages commenting out code actually. I think you can just delete it if you don't plan to use/keep it.